src/modules/keyboardmanager/ui/SingleKeyRemapControl.cpp

```diff
@@ -58,15 +58,15 @@ void SingleKeyRemapControl::createDetectKeyWindow(IInspectable const& sender, Xa
 {
     // ContentDialog for detecting remap key. This is the parent UI element.
     ContentDialog detectRemapKeyBox;
-
+    
+    // TODO: Hardcoded light theme, since the app is not theme aware ATM.

```

Comment:
@traies : Is there a task to account for making this 'theme aware'?

Reply:
@arjunbalgovind has a task about making the colors of the UI match the spec, but it does not mention Dark mode.